lib/mail_room/configuration.rb

```diff
@@ -2,16 +2,21 @@ module MailRoom
   # Wraps configuration for a set of individual mailboxes with global config
   # @author Tony Pitale
   class Configuration
-    attr_accessor :mailboxes, :daemonize, :log_path, :pid_path
+    attr_accessor :mailboxes, :daemonize, :log_path, :pid_path, :quiet
 
     # Initialize a new configuration of mailboxes
     def initialize(options={})
       self.mailboxes = []
+      self.quiet = options.fetch(:quiet, false)
 
       if options.has_key?(:config_path)
-        config_file = YAML.load_file(options[:config_path])
+        begin
+          config_file = YAML.load_file(options[:config_path])
 
-        set_mailboxes(config_file[:mailboxes])
+          set_mailboxes(config_file[:mailboxes])
+        rescue => e
```

Comment:
That only handles the two specific cases you're worried about. I'd like to handle other cases in configuration. To explicitly check with guard statements for every one of those cases does not seem cleaner to me. Also, not a fan of guard statements, especially stacked ones.